### PR TITLE
Add Ovio.org to open-source contribution initiatives

### PR DESCRIPTION
**Pull request by ericboucher**
Add https://ovio.org to the list of open-source contribution initiatives.

### CODE REVIEW

**Comment by ericboucher** [comment]
Fixes #418

File: README.md [hunk 1]
```diff
@@ -137,6 +137,7 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 - [First Contributions](https://firstcontributions.github.io/) - Make your first Open Source contribution in 5 minutes. A tool and tutorial to help beginners get started with contributions. [Here](https://github.com/firstcontributions/first-contributions) is the GitHub source code for the site and opportunity to make a contribution to the repository itself.
 - [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - A program to encourage Open Source contributions. Earn gifts like t-shirts and stickers by making at least 4 pull requests in the month of October.
 - [24 Pull Requests](https://24pullrequests.com) - 24 Pull Requests is a project to promote Open Source collaboration during the month of December.
+- [Ovio](https://ovio.org) - A platform with a curated selection of contributor-frienldy projects. It has a powerful issue search [tool](https://ovio.org/issues) and lets you save projects and issues for later!
```
Comment (erictleung) [inline README.md:140]:
```suggestion
- [Ovio](https://ovio.org) - A platform with a curated selection of contributor-friendly projects. It has a [powerful issue search tool](https://ovio.org/issues) and let's you save projects and issues for later.
```

**Comment by erictleung** [comment]
@ericboucher thanks for contributing! Just a few typos and spelling changes.

**Comment by ericboucher** [comment]
Nice catch, sorry! And thanks for the improvements :)

**Review by erictleung — APPROVED** [review]

**Comment by erictleung** [comment]
@ericboucher thanks for taking time to add this resource! ✨

**Pull request merged**